com.mudeprolinux.yml

```diff
@@ -0,0 +1,32 @@
+app-id: com.mudeprolinux.whakarere
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: whakarere
+
+finish-args:
+  # X11 access
+  - --socket=fallback-x11
```

Comment:
share=ipc is needed as I previously mentionned.

Reply:
added ipc support again